### DISCUSSION TITLE
Facing 302 error

### DISCUSSION THREAD

**Discussion by Ali1094** [osiset/laravel-shopify#721] [General]
Hi
I'm facing a serious issue when I installed my app than it installs correctly but when I create a form and send post request using ajax on '/' routes that it would be also fine and form successfully send to the controller but when I redirect to another route named '/abc' and repeating the same process than it's giving me 302 redirect issue. When I start debugging it than I see some changes in routes Example:
**https://new-laravel.test/?hmac=337da5186be47893c7ccba84f108cead07100d81593ff1dea2698a43786d47c0&locale=en&new_design_language=true&session=14a84434e9d430fadf8c0cbb2860f7c8d58b6f73015a568e719bf7284f96c917&shop=gallery-one-shop.myshopify.com×tamp=1614959201** 
but my new route is something like this
**https://new-laravel.test/bundles** which is giving 302

I don't know what's the reason behind this. Please help me out.

**Comment by xmartiiix** [comment]
Hi, 
Please try to add @sessionToken inside your form. 
https://github.com/osiset/laravel-shopify/wiki/Authentication-Process
